### ISSUE TITLE
cooler cload does not accept .lz4-compressed .pairs files

### ISSUE THREAD

**Issue by bskubi** [open2c/cooler#410]
The pairtools library encourages use of .lz4 compression, but I get an `invalid start byte` error if I try to use .lz4-compressed .pairs as input to `cooler cload pairs`. This error goes away if I decompress the .lz4 to plaintext and use that instead. The CLI documentation for `cooler cload pairs` doesn't state which type of compression it accepts, but does say "Accepts compressed files." I'd recommend either supporting .lz4 or clarifying which types of compression it works with.

**Comment by nvictus** [comment]
Good catch. We specifically support the compression that [pandas.read_csv supports](https://pandas.pydata.org/docs/user_guide/io.html#quoting-compression-and-file-format), which does not include lz4.
